pkg/expect/expect_test.go

```diff
@@ -18,12 +18,25 @@ package expect
 
 import (
 	"os"
+	"os/exec"
 	"testing"
 	"time"
 )
 
+func findBinaryOrSkip(t *testing.T, binName string) string {
+	t.Helper()
+
+	if binPath, _ := exec.LookPath(binName); binPath != "" {
```

Comment:
Why do you ignore the returned error?  Note that exec.LookPath may fail due to PathError or PermissionError, so it would be better to check the returned error, and print it in the log.

Reply:
It would - I didn't in the first pass here because we didn't check anything before. I can update this later.